src/group/example-a.njk

```diff
@@ -33,6 +33,7 @@ css:
   withBrowser(async ({
     screen, utils, page, user
   }) => {
+
```

Comment:
This makes the spacing consistent in this code block (blank line before each comment)